package-lock.json

```diff
@@ -12590,7 +12569,7 @@
     },
     "probot-config": {
       "version": "git+https://github.com/bobvanderlinden/probot-config.git#9825185ee0a1698120193ad34e6832bc43f90e18",
-      "from": "git+https://github.com/bobvanderlinden/probot-config.git#9825185ee0a1698120193ad34e6832bc43f90e18",
+      "from": "git+https://github.com/bobvanderlinden/probot-config.git#pr-probot-7.4.0",
```

Comment:
I'm not sure whether this is intended, but this change seems unrelated to the PR?

Reply:
Err, no idea how that happened. Sorry.

Reply:
My guess is that came from me running `npm it` or similar (it's really great that projects have such a verb, it's such a pain that the verb causes churn).